packages/yoroi-extension/app/api/ada/lib/state-fetch/remoteFetcher.js

```diff
@@ -407,4 +410,23 @@ export class RemoteFetcher implements IFetcher {
       return res;
     }, {});
   }
+
+  getCatalystRoundInfo: CatalystRoundInfoRequest => Promise<CatalystRoundInfoResponse> = (body) => {
+    const { BackendService } = body.network.Backend;
+    if (BackendService == null) throw new Error(`${nameof(this.getCatalystRoundInfo)} missing backend url`);
+    return axios(
+      `${BackendService}/api/v0/catalyst/fundInfo`,
```

Comment:
The `/api` prefix in the endpoint is conditional and only exists on the testnet. Must add additional check to not add it in case it's mainnet. See example in the mobile:

https://github.com/Emurgo/yoroi-mobile/blob/758a030ef584c18cccc266b2b65e152666890bbd/src/api/shelley/api.js#L176-L179